File: PythonCode/1060.py
a = float(input())
b = float(input())
c = float(input())
d = float(input())
e = float(input())
f = float(input())

qtde = 0

if a > 0:
    qtde += 1
if b > 0:
    qtde += 1
if c > 0:
    qtde += 1
elif d > 0:
    qtde += 1
if e > 0:
    qtde += 1
if f > 0:
    qtde += 1
print('{} valores positivos'.format(qtde))

File: C++Code/1060.cpp
//Questão: Numeros Positivos
#include <iostream>
#include <vector>

using namespace std;

int main(){
    float n,qtde;
    vector<float> v;
    for(float i = 0; i < 6; ++i){
        cin >> n;
        v.push_back(n);
        
        if (v[i] > 1)
        {
          qtde += 1;
        }
    }
    cout<<qtde<<" valores positivos\n";
    return 0;
}